proto/config/netconfig.proto

```diff
@@ -181,6 +187,16 @@ message CellularAccessPoint {
   // Not listed technologies will not be tried.
   // If empty, then modem will select RAT automatically.
   repeated org.lfedge.eve.common.RadioAccessTechnology preferred_rats = 7;
+  // The IP addressing type to use for the default bearer.
+  org.lfedge.eve.common.CellularIPType ip_type = 8;
```

Comment:
Is there a wildcard value for this which will cause the existing behavior (which is to try all until one works)?

Reply:
The zero value that EVE will use when upgraded from an older version is `CELLULAR_IP_TYPE_UNSPECIFIED`.
Newer EVE will also use it when user does not specify the IP type.
I added comment for this value explaining that EVE will try every possible IP type in that case.

Reply:
I found out that the ip-type does not have to be specified in the bearer config, in which case the modem will use the pre-installed profiles to determine the right IP type for the network. I think this is better behavior going forward. I have seen many problems with our current approach of trying to connect with each IP-type with some timeout. It is problematic mostly because we do not know what is the ideal timeout. If it is too much, we are significantly delaying connectivity when trying wrong ip-type, and if it is too low, we may fail even with correct IP-type.
So now that it is configurable, I would keep the field as unspecified when not configured by the user and let the modem determine the proper value.
I have updated the comment for `CELLULAR_IP_TYPE_UNSPECIFIED`.